vNext/src/NuGet/Microsoft.Orleans.OrleansAzureUtils.nuspec

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Orleans.OrleansAzureUtils</id>
+    <version>$version$</version>
+    <title>Microsoft Orleans Azure Utilities</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,Orleans</owners>
+    <projectUrl>https://github.com/dotnet/Orleans</projectUrl>
+    <licenseUrl>https://github.com/dotnet/Orleans#license</licenseUrl>
+    <iconUrl>https://raw.githubusercontent.com/dotnet/orleans/gh-pages/assets/logo_128.png</iconUrl>
+    <summary>
+      Azure Utilities Library of Microsoft Orleans - OrleansAzureUtils.dll
+    </summary>
+    <description>
+      Library of utility types for Microsoft Azure of Microsoft Orleans.
+    </description>
+    <copyright>Copyright Microsoft 2015</copyright>
+    <tags>Orleans Cloud-Computing Actor-Model Actors Distributed-Systems C# .NET</tags>
+    <dependencies>
+      <group targetFramework=".NETStandard1.5">
+        <dependency id="Microsoft.Orleans.Core" version="$version$" />
+        <dependency id="Microsoft.Orleans.OrleansRuntime" version="$version$" />
+        <dependency id="Microsoft.Orleans.OrleansProviders" version="$version$" />
+        <dependency id="WindowsAzure.Storage" version="7.2.1" />
+        <dependency id="System.Net.Requests" version="4.0.11" />
+      </group>
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="OrleansAzureUtils.dll" target="lib\netstandard1.5" />
+    <file src="OrleansAzureUtils.pdb" target="lib\netstandard1.5" />
+    <file src="OrleansAzureUtils.xml" target="lib\netstandard1.5" />
+    <file src="$SRC_DIR$OrleansAzureUtils\**\*.cs" target="src" />
```

Comment:
If possible, the src files which are in the obj & bin folders should be explicitly excluded using `exclude="$SRC_DIR$OrleansAzureUtils\obj\**\*.cs;$SRC_DIR$OrleansAzureUtils\bin\**\*.cs;". I'm not sure if this will cause issues, but just in case.